katello/hooks/pre/30-upgrade.rb

```diff
@@ -35,8 +35,9 @@ def migrate_foreman
 
 def postgresql_10_upgrade
   stop_services
+  locale = `locale | grep LANG`.split('LANG=')[1].strip
```

Comment:
We enforce `en_US.utf8` in [puppet-foreman](https://github.com/theforeman/puppet-foreman/blob/36aa5389b7da41d4770fa61a1c10a78239695735/manifests/database/postgresql.pp#L25) and [puppet-candlepin](https://github.com/theforeman/puppet-candlepin/blob/2f08eeb92c96a0c2bca3e381780cc58530467459/manifests/database/postgresql.pp#L42). Perhaps safer to do the same here?

Reply:
fwiw, the failing DB is `postgres`, not `foreman` or `candlepin`, and we don't create that.